README.md

```diff
@@ -13,45 +24,30 @@ The validating admission controller [requires](https://kubernetes.io/docs/refere
 
 Note: If you can't use the validation webhook, pass the `--disable-admission-controller` to the operator (like in `deploy/operator/basic/deployment.yaml`).
 
-## Spinnaker installed in under a minute (or two)
+## Quick Start
 
 This is a high-level view of the commands you need to run for those who want to jump right in. More explanation can be found in the sections after this one.
 
 ```bash
-# For a stable release (https://github.com/armory/spinnaker-operator/releases)
+# Pick a release from https://github.com/armory/spinnaker-operator/releases (or clone the repo and use master branch for the latest development work)
 $ mkdir -p spinnaker-operator && cd spinnaker-operator
-$ RELEASE=v0.2.0 bash -c 'curl -L https://github.com/armory/spinnaker-operator/releases/download/${RELEASE}/manifests.tgz | tar -xz'
+$ RELEASE=v0.3.0 bash -c 'curl -L https://github.com/armory/spinnaker-operator/releases/download/${RELEASE}/manifests.tgz | tar -xz'
  
-# For the latest development work (master) 
-$ git clone https://github.com/armory/spinnaker-operator.git && cd spinnaker-operator
-
 # Install or update CRDs cluster wide
 $ kubectl apply -f deploy/crds/
 
 # Install operator in namespace spinnaker-operator, see below if you want a different namespace
 $ kubectl create ns spinnaker-operator
 $ kubectl -n spinnaker-operator apply -f deploy/operator/cluster
 
-# Install Spinnaker in "spinnaker" namespace
+# Install Spinnaker in "spinnaker" namespace, connected to S3 bucket "spinnaker" (bucket must already exist and be accessible from Kubernetes EKS IAM role of the worker nodes)
```

Comment:
I'd remove comments about EKS as it sounds like a requirement to use the operator. Can we word it differently? Perhaps: `Note that this configuration uses an S3 bucket that Spinnaker needs to be able to access (e.g. on EKS, you can grant access to the worker node on which Spinnaker runs).